main.css

```diff
@@ -0,0 +1,103 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap');
+
+:root{
+    --primary-color :#F65314;
+    --secondary-color :black;
+    --tertiary-color:#8da9bb;
+    --dark-color :#E50914; 
+    --light-color: white;
+    --yellow-shade-color :#FF9900;
+    --yellow-shade-color2: #ff7b25;
```

Comment:
perhaps the name of **yellow-shade-color2** can be named **yellow-shade-light** if the purpose of the variable is to provide light color